index.html

```diff
@@ -25,6 +25,25 @@ <h1 id="logo">Aziz Mejri</h1>
           </li>
         </ul>
       </nav>
+      <nav id="mobile-nav" class="mobile-nav">
+        <div class="mobile-nav-container">
+          <img src="media/svg/image_geometry_menu_1.svg" alt="design-up" class="nav-design-up">
+          <button type="button" class="menu-close">
+            <img src="media/svg/close_icon.svg" alt="close_icon" class="close_icon">
+          </button>
+          <ul>
+            <h2 class="mobile-logo">Aziz Mejri</h2>
+            <li><a class="nav-link" href="#projects">Portoflio</a></li>
+            <li><a class="nav-link" href="#about-me">About</a></li>
+            <li>
+              <a class="nav-link" href="#contact"
```

Comment:
As pointed out by the previous reviewer, kindly reduce the font size so the project navbar will look more similar to the original one.